src/components/LibraryInfo/index.js

```diff
@@ -13,19 +13,35 @@ governing permissions and limitations under the License.
 import libraryVersion from "../../constants/libraryVersion";
 import { CONFIGURE, SET_DEBUG } from "../../constants/coreCommands";
 
+const prepareLibraryInfo = ({ config, componentRegistry }) => {
+  const allCommands = [
+    ...componentRegistry.getCommandNames(),
+    CONFIGURE,
+    SET_DEBUG
+  ].sort();
+  const resultConfig = { ...config };
```

Comment:
I didn't catch this before, but it looks like in the previous version you could modify the config object by calling getLibraryInfo.